config/app.default.php

```diff
@@ -244,6 +249,8 @@
              * which is the recommended value in production environments
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
+            
+            'url' => env('DATABASE_URL', ''),
```

Comment:
[isset()](https://github.com/cakephp/cakephp/blob/master/src/Core/StaticConfigTrait.php#L104) is used not empty().